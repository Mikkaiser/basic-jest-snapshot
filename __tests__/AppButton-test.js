import React from 'react';
import AppButton from '../src/components/AppButton';
import renderer from 'react-test-renderer';

test('AppButton renders correctly', () => {
    const tree = renderer.create(<AppButton onPress={() => alert('Hello World')}/>).toJSON();
    expect(tree).toMatchSnapshot();
})