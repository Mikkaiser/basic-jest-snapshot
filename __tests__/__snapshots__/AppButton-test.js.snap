// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppButton renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#121212",
      "borderRadius": 25,
      "justifyContent": "center",
      "opacity": 1,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#fff",
      }
    }
  >
    Register
  </Text>
</View>
`;
